src/server/status_server.rs

```diff
@@ -16,6 +17,7 @@ use std::str::FromStr;
 
 use super::Result;
 use crate::config::TiKvConfig;
+use pprof;
```

Comment:
Prefer putting all 3rd dependencies together, although the previous imports ignore the rules.